admin_guide/cns_recommendations.adoc

```diff
@@ -0,0 +1,52 @@
+### CNS - Container Native Storage recommendations for OCP
+
+OCP platform offers CNS storage feature which makes easier for OCP users to fulfill their
+storage needs. With CNS solution users and admins are empowered to have together storage and
+application pods running together on same infrastructure and sharing same resources,this leads
+to better resource utilization.
+Follow CNS https://access.redhat.com/documentation/en-us/red_hat_gluster_storage/3.1/html-single/container-native_storage_for_openshift_container_platform_3.4/[Documentation]
+for instructions how to configure CNS as part of OCP cluster.
+
+Per CNS documentation currently it is supported up to 100 CNS volumes
+on one Trusted Storage Pool ( TSP )
+
+==== CNS volumes create time
+When deploying applications on OCP it is important to estimate how long
+it will take to build environment from storage side in order for an application pod to able to start.
+If application pod requires an PVC (Persistent Volume Claim) then at below graph
+is presented how long it takes for different number of CNS to be created and bound to corresponding PVC.
+All tests were performed on one TSP (Trusted Storage Pool) and using hardware configuration for
+CNS per CNS https://access.redhat.com/documentation/en-us/red_hat_gluster_storage/3.1/html-single/container-native_storage_for_openshift_container_platform_3.4/[Documentation]
+
+Dynamic storage provisioning and storage classes were aslo configured and used when provisioning PVC
+
+.Time to create different number of CNS volumes
+image::https://raw.githubusercontent.com/ekuric/openshift/master/cns/images/create_10_50_100.png[Create CNS volumes]
```

Comment:
have we been including performance graphs anywhere else in our docs? i dont think so -- i wonder if we can just phrase this as "creation time scales linearly up through 100 volumes.  in our tests, each volume takes approximately 6 seconds to be created, allocated and bound to a pod."